tools/windows/win_compile_time_check.ps1

```diff
@@ -0,0 +1,36 @@
+# Script to test if a small target builds quickly on windows as expected.
+
+# Define the timeout value in seconds
+$timeout = 120
+
+$folderPath = ".\out\win32_test\"
+
+# Remove folder if present from out directory
+if (Test-Path $folderPath) {
+    # Delete the folder
+    Remove-Item $folderPath -Recurse -Force
+}
+
+# Start a timer
+$start = Get-Date
+
+# Run the command
+$command = "python .\cobalt\build\gn.py .\out\win32_test -p win-win32 -C devel ; ninja -C $folderpath eztime_test"
+Invoke-Expression $command
+
+# Get the end time
+$end = Get-Date
+
+# Calculate the execution time
+$executionTime = $end - $start
+$executionTimeinSecs = $executionTime.seconds

```

Comment:
I just tested this and it turns out $executionTime.seconds returns the seconds "digit" of the time, not the number of seconds elapsed. So if it runs for 70 seconds, this will be 10.